draft/contents/profile.md

```diff
@@ -17,9 +17,11 @@ Der Link zum JSON-LD-Kontext für das Dokument MUSS auf `https://schema.org/` ge
     <dt>Pflichtfeld</dt>
```

Comment:
Mir fällt gerade auf, dass wir beim Kontext nur die oberste Hierarchie-Ebene derzeit berücksichtigen, sollten wir das nicht entsprechend anpassen?
https://github.com/dini-ag-kim/lrmi-profile/blob/master/draft/schemas/context-schema.json

Reply:
Ja, du hast recht. Die Spec stimmt hier noch nicht mit dem Schema überein.

Reply:
Gefixt in df3b102. Schau bitte mal drauf.